doc/src/fix_pimd.rst

```diff
@@ -149,6 +149,31 @@ related tasks for each of the partitions, e.g.
    restart 1000 system_${ibead}.restart1 system_${ibead}.restart2
    read_restart system_${ibead}.restart2
 
+Restart, fix_modify, output, run start/stop, minimize info
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+This fix writes the state of the Nose/Hoover thermostat over all quasi-beads to
```

Comment:
Thanks, @akohlmey, for explaining how the restart files work on LAMMPS.
In that context, do you think this phrasing is adequate?

Reply:
I will look at it later today. If needed, I will make some final changes myself.
I appreciate you both taking the time to add the missing text, even for the parts you didn't write. It is a big help.